bson-bench/src/common.ts

```diff
@@ -0,0 +1,199 @@
+import * as cp from "child_process";
+import { once } from "events";
+import * as fs from "fs/promises";
+import * as path from "path";
+
+// TODO: Should allow latest, betas and alphas
```

Comment:
take care of this

Reply:
Should we make the regex here looser and let npm handle any errors?
As an extension of this, should we capture the stderr from calling npm install? I can see that it might help with debugging.